documentation/asciidoc/computers/linux_kernel/building.adoc

```diff
@@ -80,6 +80,13 @@ KERNEL=kernel7l
 make bcm2711_defconfig
 ----
 
+For Raspberry Pi 4 default build configuration (64 bit)
+[,bash]
+----
+cd linux
+KERNEL=kernel8
+make bcm2711_defconfig
+
 ===== Customising the Kernel version using `LOCALVERSION`
```

Comment:
`Kernel` -> `kernel`

Reply:
I've not changed this because there are lots of headings here with capitalized Kernel.  @aallan should I remove all the mixed capitalization from headings (there's a mix-mash)

Reply:
We generally follow the AP Style Guide, so we use [title case](https://en.wikipedia.org/wiki/Title_case) in section headings. Kernel **should** be capitalised in this instance.

Reply:
…but so should "version" and "using".